src/components/tools/WindowLevelTool.vue

```diff
@@ -149,18 +134,18 @@ const WindowLevelToolComponent = defineComponent({
         horizRange.step,
         () => horizVal.value,
         (v) => {
-          horizVal.value = v;
+          if (currentImageID.value != null) {
+            windowingStore.updateConfig(viewID.value, currentImageID.value, {
+              level: v,
+            });
+          }
         }
       );
     }
 
     watch(
-      () => [wwRange.value, wlRange.value],
-      ([ww, wl]) => {
-        vertVal.value = ww.default;
-        horizVal.value = wl.default;
-        updateManipulator();
-      },
+      () => JSON.stringify({ w: wwRange.value, l: wlRange.value }),
```

Comment:
I think we can simplify it to be:
```
watch([wwRange, wlRange], () => ..., { deep: true, immediate: true })

Reply:
> `watch([wwRange, wlRange], () => ..., { deep: true, immediate: true })`

Still calls `updateManipulator` every window/config update.  I think because the `wwRange` ref is a fresh object with every update to `wlConfig`.

Reply:
Ah, I see, you're going for a deep equals then. Looks good to me! I'd recommend making a comment on this line explaining this.

Reply:
Comment added.